portfolio/src/main/java/com/google/sps/servlets/LoginAPI.java

```diff
@@ -44,11 +44,13 @@
     boolean isLoggedIn; 
     String loginLink; 
     String logoutLink; 
+    String userEmail; 
 
-    public LoginInfo(boolean isLoggedIn, String loginLink, String logoutLink) {
+    public LoginInfo(boolean isLoggedIn, String loginLink, String logoutLink, String userEmail) {
```

Comment:
why not just call it userName. in the JS file we seem to call it userName

Reply:
I ended up wanting to keep track of the user's email and the name they write to display to the screen (userName), so I kept track of them with two different variable names.